composer.json

```diff
@@ -16,5 +16,8 @@
     },
     "autoload": {
         "classmap": ["lib"]
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7"
```

Comment:
this breaks Travis for php < 5.6. I know these PHP versions are really old by now, but some people are still using them, so it would be good if we could avoid a breakage here (otherwise we'd have to bump the major version for a release)